recipe/meta.yaml

```diff
@@ -12,21 +12,20 @@ source:
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-build-isolation -vvv
+  skip: True # [py<37]
 
 requirements:
   host:
     - pip
-    - aiohttp
-    - tornado >=6.0
-    - pymongo >=4.1,<5
-    - python >=3.7
+    - python
+    - wheel
+    - setuptools
   run:
     - aiohttp
-    - tornado >=6.0
-    - pymongo >=4.1,<5
-    - python >=3.7
+    - tornado
```

Comment:
`tornado` isn't mentioned in the requirements https://github.com/mongodb/motor/blob/19eeb29b98fac159952b6a0c3440c91d77ee276a/pyproject.toml#L46. But it used as a framework here https://github.com/mongodb/motor/blob/3.4.0/motor/frameworks/tornado/__init__.py#L25-L28. Could you please add a comment about the usage in the source code (for our future selves)

Reply:
added the comment!